frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx

```diff
@@ -189,10 +237,31 @@ export const IndividualResponsePage = (): JSX.Element => {
           </Text>
           <Box display="inline-flex">
             <Text as="span" textStyle="subhead-1">
-              Time:&nbsp;
+              Timestamp:
+            </Text>
+            <Skeleton isLoaded={!isLoading && !isError}>
+              &nbsp;{data?.submissionTime ?? 'Loading...'}
+            </Skeleton>
+          </Box>
+          <Box
+            alignItems="center"
+            display="inline-flex"
+            hidden={attachmentDownloadUrls.size === 0}
```

Comment:
i think we should not even render if there are no attachments instead of just hiding this section